### DISCUSSION TITLE
4 base stations in the lighthouse tracking system do not support 5 crazyfiles

### DISCUSSION THREAD

**Discussion by Guohao23** [bitcraze/discussions#1289] [Q&A]
Hello everyone,
I used 5 Crazyflies in the lighthouse tracking system. The lighthouse tracking system uses four base stations, and I followed all the instructions detailed in the Lighthouse Start Guide and made appropriate changes to the hover_swarm.launch file. However, when I started the above file and executed chooser. py, there were only a maximum of three crazyfiles that could be displayed in rviz. When selecting more than 3, only two appear in rviz. The above issue did not occur when using optitrack before.

**Comment by knmcguire** [comment]
This is not an issue with the lighthouse positioning system but this sounds like an issue in of Crazyswarm1. 

Either way, this question should be asked on:
- Crazyswarm 1: https://github.com/USC-ACTLab/crazyswarm/discussions (support is quite slow as this project has been superseded by crazyswarm2.)
- Crazyswarm 2:  https://github.com/IMRCLab/crazyswarm2/discussions
